MigrationTutorial/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using MigrationTutorial.Services;
+using MigrationTutorial.Utils;
+using Realms;
+
+namespace MigrationTutorial
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            try
+            {
+
+#region Parameter parsing
+                var cmdParams = new Dictionary<string, string>();
```

Comment:
Initially I was allowing to specify `--delete_realm` but I then removed it as it made no much sense to delete the Realm any other time than when "restarting from scratch" (= schemaVersion 1). After that I still left the parsing part. 
I also left it there since, this work being still a draft, you may have had suggestions on changing some of the command line behaviour/interface.
If you think it's confusing since there's only 1 arg then I'll remove it.

Reply:
I think we can keep it for now, in case you want to bring back multiple args, but at the end I'd remove it if we still have only one.